cli/cmd/stat.go

```diff
@@ -284,6 +305,29 @@ func writeStatsToBuffer(rows []*pb.StatTable_PodGroup_Row, w *tabwriter.Writer,
 				tcpWriteBytes:      getByteRate(r.GetTcpStats().GetWriteBytesTotal(), r.TimeWindow),
 			}
 		}
+		if r.TsStats != nil {
+			leaf = r.TsStats.Leaf
```

Comment:
I think we can make these three `:=` declarations and remove the global variables above.